### ISSUE TITLE
Rattus norvegicus RGSC 6.0/rn6 genome database

### ISSUE THREAD

**Issue by alussier17** [shenlab-sinai/region_analysis#29]
Hello,

I would like to use your software on sequencing data aligned to the newest R. norvegicus genome (RGSC 6.0/rn6). Could you please add it to your annotation database?

Cheers,

alussier17

**Comment by ny-shao** [comment]
Hi alussier17,

I'm sorry but unfortunately the current Ensembl is still old rn5 version.

[ftp://ftp.ensembl.org/pub/release-78/gtf/rattus_norvegicus/](ftp://ftp.ensembl.org/pub/release-78/gtf/rattus_norvegicus/)

region_analysis need the biotype information from Ensembl, so now we can't add rn6. I will keep this thread open and when Ensembl update their database I promise I will add it.

Best,
Ning-Yi

**Comment by alussier17** [comment]
Hi Ning-Yi,

That's unfortunate... thank you for letting me know and for your quick reply. Hopefully, it will be up relatively soon. 

Cheers,

alussier17

**Comment by VivekTodur** [comment]
Hi,

Now it is available. Could you please add Rno6. Or at least can you explain how to prepare the database...??

thanks,
Vivek

**Comment by subhoshreeghose24** [comment]
Hi,

I am also trying to perform region analysis with my DMR file (diff.nb.txt format) but getting an error of unsupported genome. So it would be really kind if you add this to the default browser.  I tried with the refFlat rn6 file downloaded from ucsc but it is not supporting...


Thanks 
Subhoshree

**Comment by VivekTodur** [comment]
Hi Subhoshree,

Once you download the refFlat, just need to reorganize the columns and the example for the same is here,

**Vom2r1	NM_001099461	1	-	245887	267011	245887	267011	7	245887,256156,257489,258566,259804,260546,266805,	246774,256280,257714,259370,260077,260562,267011,**

But the only thing I am not sure is the way to define the path for a newly created database, kindly share if you figured out this.

Thanks
Vivek

**Comment by subhoshreeghose24** [comment]
okay I will definitely try this...But the issue was initially that when I am running the command with( .txt) file and a ( -g) before that it is always showing unsupported genome.....

**Comment by VivekTodur** [comment]
Yeah, rn6 is basically not supported. So try with rn4 it should work fine. If it works fine then next challenge is to prepare the database and storing in the right path, where the tool is capable of reading it.

Vivek

**Comment by subhoshreeghose24** [comment]
okay...So you are suggesting I should run my diffReps output file
(diff.nb.txt) which has been done with rn6 with rn4 region analysis and
later on liftover the coordinates using ucsc.....I will try that then

Subhoshree Ghose
CSIR-SRF
C/O Dr. Shantanu Sengupta
Lab No. 201
CSIR-IGIB
